ern-cauldron-api/src/CauldronApi.ts

```diff
@@ -1173,7 +1173,7 @@ export default class CauldronApi {
     if (fileName) {
       const pathToOldYarnLock = this.getRelativePathToYarnLock(fileName)
       await this.fileStore.removeFile(pathToOldYarnLock)
-      const newYarnLockFileName = shasum(yarnlock)
+      const newYarnLockFileName = uuidv4()
```

Comment:
if we are not using `shasum` we can remove the code from the `uitls.js`

Reply:
yes, good call, was the only use left. 
let me remove it.